keyboards/duck/octagon/v2/rules.mk

```
@@ -15,8 +15,8 @@ BOOTLOADER = atmel-dfu
 #   change yes to no to disable
 #
 BOOTMAGIC_ENABLE = lite       # Virtual DIP switch configuration
-MOUSEKEY_ENABLE = no        # Mouse keys
-EXTRAKEY_ENABLE = no        # Audio control and System control
+MOUSEKEY_ENABLE = yes        # Mouse keys
+EXTRAKEY_ENABLE = yes        # Audio control and System control
```

```suggestion
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = no       # Audio control and System control
```
You should leave these as default, and add both changes to the `rules.mk` in your own keymap folder: `keyboards/duck/octagon/keymaps/chilok29/rules.mk`. See [this page](https://docs.qmk.fm/#/newbs_building_firmware) for more details.